pkg/provider/aws/kv.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/klothoplatform/klotho/pkg/core"
+	"github.com/klothoplatform/klotho/pkg/provider/aws/resources"
+)
+
+func (a *AWS) GenerateKvResources(kv *core.Kv, result *core.ConstructGraph, dag *core.ResourceGraph) error {
```

Comment:
so how are we going to handle this with multiple annotations with different IDs for kv, because this would essentially make us loop over those and create a table per, which i believe is the correct method, but we likely need to handle that in some regard

Reply:
yeah youll just need to add the new reference tothe atable i think because it will only ever have the first kv as a ref and then nothing else